.eslintrc

```diff
@@ -10,7 +10,7 @@
     "node": true,
     "browser": true
   },
-  "extends": "eslint:recommended",
+  "extends": "@atlassian/atlassian-fecq",
```

Comment:
We've decided to just use the AirBnb eslint rules in AtlasKit. They're much more stringent and closer to what we want anyways, I think. Maybe use those?